### DISCUSSION TITLE
Which models are the best balance between speed, VRAM usage and quality?

### DISCUSSION THREAD

**Discussion by MoltenMonster** [ggml-org/whisper.cpp#1480] [Q&A]
I'm impressed by how fast Whisper.cpp runs on my M1 Mac. I gave it a 20 min recording and the tiny one did it in 35 secs, while the base one took 40 secs. I am wondering how the quality is with the larger models and if its worth the extra RAM (M1 unified RAM) and less speed.

**Comment by bobqianic** [comment]
- 38M -> Tiny
- 73M -> Base
- 244M -> Small
- 768M -> Medium
- 1549M -> Large

![image](https://github.com/ggerganov/whisper.cpp/assets/129547291/ee3ef007-2b02-4c93-99e9-09b4fcf92d5b)

**Reply by YousufSSyed** [reply]
@bobqianic Where did you get this from? I did some looking up for OpenAI Whisper WER scores but didn't find any other than this one.

**Reply by bobqianic** [reply]
> Where did you get this from?

https://github.com/openai/whisper/discussions/661
